server/public/views/signup.html

```diff
@@ -74,170 +74,171 @@ <h1 class="section-heading">Sign Up</h1>
-                            <p class="h4 col-sm-10 col-sm-offset-1 text-thin">
-                                Enter the addresses and an address code (e.g. home, her work, her place, etc) for each location 
-                                you might want her gifts delivered to. Address codes are used as a way to easily enter the delivery 
-                                address when ordering gifts by text.
-                            </p>
-                        </div>
-                        <div class="col-xs-12 top-20">
-                            <div class="row">
-                                <div class="col-sm-6">
-                                    <div class="form-group">
-                                        <label class="control-label">Addresses One*</label>
-                                        <input type="text" class="form-control" id="s-addressOne" placeholder="Address" required />
-                                        <small class="text-danger" id="s-addressOneError" style="display: none;"></small>
-                                    </div>
-                                    <div class="form-group">
-                                        <label class="control-label">Suite/Apt</label>
-                                        <input type="text" class="form-control" id="s-addressOneSuite" placeholder="Suite or Apt# (e.g. Suite D, Apt 245, #104)" maxlength="15" />
-                                        <small class="text-danger" id="s-addressOneSuiteError" style="display: none;"></small>
-                                    </div>
-                                    <div class="form-group">
-                                        <label class="control-label">Addresses One - Code Name*</label>
-                                        <input type="text" class="form-control" id="s-addressOneCode" placeholder="Address Code Name (e.g. home, work)" required />
-                                        <small class="text-danger" id="s-addressOneCodeError" style="display: none;"></small>
-                                    </div>
-                                    <div class="form-group">
-                                        <label class="control-label">Addresses One - Delivery For</label>
-                                        <input type="text" class="form-control" id="s-addressOneDeliveryFor" placeholder="Addressed To (e.g. Her Name, Your Name)" />
-                                    </div>
+                <!-- Shipping Addresses -->
+                <div class="row">
+                    <div class="col-sm-12 text-center">
+                        <hr class="primary">
+                        <h2 class="section-heading">Shipping Addresses</h2>
+                        <p class="h4 col-sm-10 col-sm-offset-1 text-thin">
+                            Enter the addresses and an address code (e.g. home, her work, her place, etc) for each location 
```

Comment:
@armandopmj doesn't the address code have to be one word? So `her work` or `her place` won't work.